help/destinations/catalog/advertising/pubmatic.md

```diff
@@ -0,0 +1,129 @@
+---
+title: PubMatic Connect
+description: PubMatic maximizes customer value by delivering the programmatic digital marketing supply chain of the future. PubMatic Connect combines platform technology and dedicated service to enhance how inventory and data are packaged and transacted.
+---
+
+# PubMatic Connect destination {#your-destination}
+
+## Overview {#overview}
+
+PubMatic maximizes customer value by delivering the programmatic digital marketing supply chain of the future. PubMatic Connect combines platform technology and dedicated service to enhance how inventory and data are packaged and transacted.
+
+This destination can be used to send audience data to the PubMatic Connect platform.
+
+> [!IMPORTANT]
+>
+> The destination connector and documentation page are created and maintained by the PubMatic team. For any inquiries or update requests, please contact them directly at `support@pubmatic.com`.
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the PubMatic Connect destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+### Use case #1 {#use-case-1}
```

Comment:
If you are not planning to add more use cases, please remove the Use case #1 heading.